daggercore/src/main/java/com/islamversity/daggercore/modules/domain/SearchRepoModule.kt

```diff
@@ -0,0 +1,36 @@
+package com.islamversity.daggercore.modules.domain
+
+import com.islamversity.core.Mapper
+import com.islamversity.db.NameQueries
+import com.islamversity.db.SurahQueries
+import com.islamversity.db.datasource.SurahLocalDataSource
+import com.islamversity.db.datasource.SurahLocalDataSourceImpl
+import com.islamversity.db.model.Surah
+import com.islamversity.domain.model.surah.SurahRepoModel
+import com.islamversity.domain.repo.SurahSearchRepo
+import com.islamversity.domain.repo.SurahSearchRepoImpl
+import dagger.Module
+import dagger.Provides
+
+
+@Module
+object SearchRepoModule {
+
+    @JvmStatic
+    @Provides
+    fun provideDataSource(
+        surahQueries : SurahQueries,
+        nameQueries : NameQueries
+    ): SurahLocalDataSource = SurahLocalDataSourceImpl(
```

Comment:
This is already provided in the DBModule

Reply:
We didn't have that when I wrote it